com.unity.ml-agents/Runtime/Policies/BarracudaPolicy.cs

```diff
@@ -45,6 +49,7 @@ internal class BarracudaPolicy : IPolicy
         {
             var modelRunner = Academy.Instance.GetOrCreateModelRunner(model, brainParameters, inferenceDevice);
             m_ModelRunner = modelRunner;
+            m_SapceType = brainParameters.VectorActionSpaceType;
```

Comment:
1) typo: m_SapceType -> m_SpaceType
2) Should we be dealing in ActionSpecs here instead? Would adding a method to BrainParameters to return an equivalent ActionSpec be helpful anywhere else?

Reply:
Could be used in creating the vector actuator.